projects/Open-Rival-test/src/tests/TestWalkerComponent.cpp

```diff
@@ -32,10 +33,63 @@ SCENARIO("WalkerComponent can plan a route", "[components][movement-component]")
             {
                 REQUIRE(!walkerComponent->getRoute().isEmpty());
                 MapNode expectedDestination = { 1, 2 };
-                REQUIRE(walkerComponent->getRoute().getDestination() == expectedDestination);
+                REQUIRE(walkerComponent->getRoute().getFinalDestination() == expectedDestination);
+            }
+        }
+    }
+    
+    GIVEN("A unit with a WalkerComponent in an enclosed space")
+    {
+        Entity e(EntityType::Unit, 1, 1);
+        e.attach(std::make_shared<WalkerComponent>());
+        e.onSpawn(&world, 0, { 3, 2 });
+
+        WalkerComponent* walkerComponent = e.requireComponent<WalkerComponent>(WalkerComponent::key);
+
+        MapNode enclosure[] = {
```

Comment:
Project convention is to put the square brackets after the type. This can also be const.

e.g.

```
const MapNode[] enclosure = ...
```

Reply:
The compiler doesn't like this one! If I move the brackets to MapNode instead, as suggested, it gives me an error. Adding const is fine though.